modules/toArray.js

```diff
@@ -16,5 +17,6 @@ export default function toArray(obj) {
     return obj.match(reStrSymbol);
   }
   if (isArrayLike(obj)) return map(obj, identity);
+  else if (isIterable(obj)) return iteratorToArray(obj[Symbol.iterator])
```

Comment:
When adding a feature to Underscore, you always also have to add [tests](https://github.com/jashkenas/underscore/tree/master/test) to verify that the feature works as intended. This is especially important because this enables us to notice when a later change breaks a pre-existing feature.

I'm mentioning this here because `_.toArray` is already a public function. If `isIterable` or `iteratorToArray` was public as well, though, you'd have to add tests for that, too.